README.md

```diff
@@ -85,6 +85,16 @@ Checkout command in the form of:
 1. Resolve with a checkout command object for the repository
 2. Reject if not able to get checkout command
 
+### getSetupCommand
+Internally calls getCheckoutCommand to get just the checkout command for a build, given a pipeline model and build configuration.l
+
+Required Parameters:
+| Parameter        | Type  | Required |  Description |
```

Comment:
Need an empty line. Right now it looks like this:

![screen shot 2016-11-16 at 5 00 30 pm](https://cloud.githubusercontent.com/assets/3401924/20372178/363c8050-ac1e-11e6-8db9-bba4e74aa1b0.png)